Triage/triage_modules/trigger_triage.py

```diff
@@ -37,6 +37,10 @@ def api_key(self):
     def api_url(self):
         return self.module.configuration["api_url"]
 
+    @property
+    def exclude_signed(self):
+        return self.configuration.get("exclude_signed", False)
```

Comment:
According to this line, It confirms that the parameter must move to the trigger configuration